swagger.yaml

```diff
@@ -0,0 +1,244 @@
+swagger: "2.0"
+info:
+  description: "discretionary-business-grants API Schema"
+  version: "0.0.1"
+  title: "discretionary-business-grants API"
+host: "discretionarybusinessgrants.hackney.gov.uk"
+basePath: "/api"
+tags:
+  - name: "urls"
+    description: "Secure URL for uploading files"
+  - name: "applications"
+    description: ""
+schemes:
+  - "https"
+paths:
+  /urls:
+    post:
+      tags:
+        - "urls"
+      summary: "Retrieve a new secure URL for file upload"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "body"
```

Comment:
*minor*: I think this endpoint might have been nicer if we used a query param instead of posting a json with 1 key.